lib/ardb.rb

```diff
@@ -39,6 +41,18 @@ def self.escape_like_pattern(pattern, escape_char = nil)
     self.adapter.escape_like_pattern(pattern, escape_char)
   end
 
+  private
+
+  # try requiring the db file via the load path or as an absolute path, if
+  # that fails it tries requiring relative to the current working directory
+  def self.require_db_file
+    begin
+      require ENV['ARDB_DB_FILE']
+    rescue LoadError
+      require File.join(ENV['PWD'], ENV['ARDB_DB_FILE'])
```

Comment:
@jcredding would it be better to use expand path here?  I've always used it when trying to get absolute paths.

`File.expand_path(ENV['ARDB_DB_FILE'], ENV['PWD'])`